### DISCUSSION TITLE
Security Onion Dashboards: how to configure queries involving double quotes

### DISCUSSION THREAD

**Discussion by logz4u** [Security-Onion-Solutions/securityonion#8294] [Unsupported Versions]
JSON newd question for escaping quotes. I want to customize some  Security Onion Dashboard (amazing feature btw) saved queries. I followed the instructions in the documentation, but I want to know how to input field values that contain double quotes (") in the dashboards.json file. For example,

>  { "name" : "Palo Alto Firewall Threats" , "description" :  "Palo Alto Threat Logs" , "query" :  event.dataset:panw.panos AND NOT panw.panos.type: "TRAFFIC" AND NOT panw.panos.type: "SYSTEM" | groupby destination.ip destination.port -sankey"} <

I know an easy workaround is not to include the filters of 'THREAT' and 'TRAFFIC', but this would be a time saver in the long run.

**Comment by btdewey** [comment]
I believe forward slashes are what your looking for, but I often leave off double quotes around the filter if it's an exact match with no wildcards.  I think there is a quote that doesn't have a closing quote just after the -sankey at the end.
escaped:
`event.dataset:panw.panos AND NOT panw.panos.type: \"TRAFFIC\" AND NOT panw.panos.type: \"SYSTEM\" | groupby destination.ip destination.port -sankey`
No quotes:
`event.dataset:panw.panos AND NOT panw.panos.type: TRAFFIC AND NOT panw.panos.type: SYSTEM | groupby destination.ip destination.port -sankey`

**Reply by btdewey** [reply]
I should have put my code in a code block at first.  The escape characters didn't show.  Sorry about that.
